woocommerce/wc-admin
# Admin note for online clothing stores

Fixes #4199

Adds an admin note that appears for fashion/apparel/accessories stores.

### Detailed test instructions:

- In the OBW
    - do not select 'setting up for a client'
    - select Fashion, apparel, and accessories on the Industry step
- note that the WCAdmin plugin needs to have been installed for at least 2 days
- Run the wc-admin-daily cron task
- The below note should appear in the inbox:

![image](https://user-images.githubusercontent.com/224531/83841387-19c24c00-a744-11ea-8d8f-06e029aff27e.png)

### Changelog Note:

Enhancement: Add admin note for online clothing stores
